_editorial_policies/04_publication_ethics.md

```diff
@@ -0,0 +1,51 @@
+- Authors are encouraged to recommend appropriate reviewers during submission of a manuscript. However, they should not suggest reviewers with whom they have an actual, perceived, or potential conflict of interest. Moreover, authors should not suggest reviewers with whom they share a personal or professional connection if there is potential for that relationship to introduce bias into the evaluation of the manuscript. 
+- Submitting authors must ensure that the listed co-authors of a paper should be all those persons who have made significant scientific contributions to the work reported and who share responsibility and accountability for the results. Authors should appropriately recognize the contributions of technical staff and data professionals. Other contributions should be indicated in a footnote or an “Acknowledgments” section. An administrative relationship to the work does not of itself qualify a person for co-authorship (but occasionally it may be appropriate to acknowledge major administrative assistance). Deceased persons who meet the criterion for inclusion as co-authors should be so included, with a footnote reporting date of death. No fictitious name should be listed as an author or coauthor. The author who submits a manuscript for publication accepts the responsibility of having included as co-authors all persons appropriate and none inappropriate. The submitting author should have sent each living co-author a draft copy of the manuscript and have obtained the co-author’s assent to co-authorship of it.
+- The corresponding author must reveal to the editor and to the readers of the journal any potential and/or relevant competing financial or other interest (of all authors) that might be affected by publication of manuscript. Conflicts of interest and sources of funding of the research reported must be clearly stated at the time of manuscript submission and will be included in the published article. In addition, all authors must declare (1) the existence of any significant financial interest (>$5,000 or >5% equity interest) in corporate or commercial entities dealing with the subject of the manuscript; (2) any employment or other relationship (within the past three years) with entities that have a financial or other interest in the results of the manuscript (to include paid consulting, expert testimony, honoraria, and membership of advisory boards or committees of the entity). The corresponding author must advise the editor at the time of submission either that there is no conflict of interest to declare, or should disclose potential conflicts of interest that will be acknowledged in the published article.
+- Images should be free from misleading manipulation. An accurate description of how the images were generated and produced should be provided.  Images that are not original must have permission to be included.
```

Comment:
"Images... must have permission" probably should be "Authors... must have permission... to include any images which are not original".

Reply:
yeah, simpler.